build_tools/packaging/linux/fetch_artifacts.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+"""Given ROCm artifacts directories, performs the download and extraction of artifacts
+
+```
+./fetch_artifacts.py --artifact-dir ./output-linux-portable/build/artifacts \
+        --gfx-arch gfx94X 
```

Comment:
Can https://github.com/ROCm/TheRock/blob/main/build_tools/fetch_artifacts.py be used instead of a new script? I'd also prefer we not have two scripts with the same name (unless they are in parallel locations like `packaging/linux/fetch_artifacts.py` and `packaging/windows/fetch_artifacts.py`).

Reply:
Updated code to use the existing script [build_tools/fetch_artifacts.py](https://github.com/ROCm/TheRock/blob/main/build_tools/fetch_artifacts.py)